templates/rag-jaguardb/rag_jaguardb/chain.py

```diff
@@ -0,0 +1,66 @@
+import os
+from langchain_community.chat_models import ChatOpenAI
+from langchain_community.embeddings import OpenAIEmbeddings
+from langchain_community.vectorstores import jaguar
+from langchain_core.output_parsers import StrOutputParser
+from langchain_core.prompts import ChatPromptTemplate
+from langchain_core.pydantic_v1 import BaseModel
+from langchain_core.runnables import (
+    RunnableParallel,
+    RunnablePassthrough,
+)
+
+if os.environ.get("JAGUAR_API_KEY", None) is None:
+    raise Exception("Missing `JAGUAR_API_KEY` environment variable.")
+JAGUAR_API_KEY = os.environ["JAGUAR_API_KEY"]
+
+
+url = "http://192.168.3.88:8080/fwww/"
+pod = "vdb"
+store = "langchain_test_store"
+vector_index = "v"
+vector_type = "cosine_fraction_float"
+vector_dimension = 1536
+embeddings = OpenAIEmbeddings()
+vectorstore = jaguar(
```

Comment:
does this work for you? I think this has to be the Jaguar class exported from that file